src/api/index.js

```diff
@@ -1,19 +1,21 @@
-export function geocodeGoogle(location) {
-  const GOOGLE_API_KEY = 'AIzaSyCs8iHIiSD7O0f15VNk5mmerXBLO-0BKsY';
+import { GOOGLE_API_KEY, BOM_API_KEY } from '../constants';
 
+export function geocodeGoogle(location) {
   const url = `https://maps.googleapis.com/maps/api/geocode/json?address=${location}&key=${GOOGLE_API_KEY}`;
 
   return fetch(url)
-    .then(res => res.json())
-    .catch(err => {});
+    .then(res => {
+      if (res.ok) return res.json();
+      throw new Error('Bad network response');
```

Comment:
I'm new to `fetch` and I got this from their docs:

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#Checking_that_the_fetch_was_successful